tensorflow/core/util/cuda_device_functions.h

```diff
@@ -28,10 +28,23 @@ limitations under the License.
 
 #include <algorithm>
 #include <complex>
-#include "third_party/eigen3/unsupported/Eigen/CXX11/Tensor"
+#ifdef PLATFORM_GOOGLE
```

Comment:
This change seems like it will break non-`PLATFORM_GOOGLE` builds, such as the Linux-with-Bazel GPU build.